packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -1170,22 +1170,39 @@ describe('e2e', function () {
         env: {
           ...process.env,
           NODE_PATH: path.resolve(__dirname, 'fixtures', 'node-path'),
+          NODE_OPTIONS: '--expose-gc',
         },
       });
       await shell.waitForPrompt();
       shell.assertNoErrors();
     });
 
-    it('require() searches the current working directory according to Node.js rules', async function () {
-      let result;
+    it('require() and import() search the current working directory according to Node.js rules', async function () {
+      let result: string;
       result = await shell.executeLine('require("a")');
       expect(result).to.match(/Error: Cannot find module 'a'/);
       result = await shell.executeLine('require("./a")');
       expect(result).to.match(/^A$/m);
       result = await shell.executeLine('require("b")');
       expect(result).to.match(/^B$/m);
+      result = await shell.executeLine('require("b-esm").value');
+      expect(result).to.match(/^B-ESM$/m);
       result = await shell.executeLine('require("c")');
       expect(result).to.match(/^C$/m);
+      result = await shell.executeLine('import("b").then(m => m.default)');
+      expect(result).to.match(/^B$/m);
+      result = await shell.executeLine('import("b-esm").then(m => m.value)');
+      expect(result).to.match(/^B-ESM$/m);
+    });
+
+    it('import() works when interleaved with GC', async function () {
```

Comment:
what is the relationship between the garbage collector and import that would cause issues?